package.json

```diff
@@ -7,10 +7,11 @@
   "author": "Marlon Keating <marlonakeating@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "webpack --config webpack.config.js && npm run build:sass && python manage.py collectstatic --noinput -i *.scss",
-    "build:sass": "node-sass civictechprojects/static/css/styles.scss -o civictechprojects/static/css/compiled --output-style compressed",
-    "flow": "flow",
-    "postinstall": "webpack --config webpack.config.js && python manage.py collectstatic --noinput -i *.scss"
+    "build": "webpack --mode development && npm run buildtask:sass && npm run buildtask:collectstatic",
+    "buildprod": "webpack --mode production && npm run buildtask:sass && npm run buildtask:collectstatic",
```

Comment:
Until we make some workflow changes in the future (watch/autorebuild, not requiring bundle as part of a branch, etc) the only workflow change is here: When developing, use `npm run build` as normal, but when ready to push a PR use `npm run buildprod` to generate a production-ready bundle.js.